src/rpm/builder.rs

```diff
@@ -707,6 +707,7 @@ impl RPMBuilder {
         }
 
         let offset = 0;
+        let small_packet;
```

Comment:
also `small_packet` -> `small_package`

Reply:
I did it to remove unnecessary unwraps later in code. Note that this variable can not be used without initialization unlike in C.

 If you want to perform calculation as early as possible, I think something like this would be better:
```rust
let (small_package, combined_file_sizes_entry) = match combined_file_sizes.try_into() {
    Ok(size) => (
        true,
        IndexEntry::new(IndexTag::RPMTAG_SIZE, offset, IndexData::Int32(vec![size])),
    ),
    Err(_) => (
        false,
        IndexEntry::new(
            IndexTag::RPMTAG_LONGSIZE,
            offset,
            IndexData::Int64(vec![combined_file_sizes]),
        ),
    ),
};
```